src/tools/analyzer/modules/error.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import logging
+
+from sssd import util
+from sssd.source_files import Files
+from sssd.source_journald import Journald
+from sssd.parser import SubparsersAction
+from sssd.parser import Option
+from sssd import sss_analyze
+
+class ErrorAnalyzer:
+    """
+    An error analyzer module, list if there is any error reported by sssd_be
+    """
+    module_parser = None
+    consumed_logs = []
+    print_opts = [
+        Option('--verbose', 'Verbose output', bool, '-v'),
+    ]
+
+    def print_module_help(self, args):
+        """
+        Print the module parser help output
+
+        Args:
+            args (Namespace): argparse parsed arguments
+        """
+        self.module_parser.print_help()
+
+    def setup_args(self, parser_grp, cli):
+        """
+        Setup module parser, subcommands, and options
+
+        Args:
+            parser_grp (argparse.Action): Parser group to nest
+               module and subcommands under
+        """
+        svc = util.Utils()
+        desc = "Analyze error check module"
+        self.module_parser = parser_grp.add_parser('error',
+                                                   description=desc,
+                                                   help='Error checker')
+
+        subparser = self.module_parser.add_subparsers(title=None,
+                                                      dest='subparser',
+                                                      action=SubparsersAction,
+                                                      metavar='COMMANDS')
+
+        subcmd_grp = subparser.add_parser_group('Operation Modes')
+        cli.add_subcommand(subcmd_grp, 'list', 'Print sssd error msg',
```

Comment:
I would change `Print sssd error msg` to `Print error messages found in backend`